README.md

```diff
@@ -21,6 +21,28 @@ You need to have `statamic/cms` installed in your Rapidez installation. Follow t
 composer require rapidez/statamic
 ```
 
+## Install command
+
+The install command will help you set up all the necessary settings.
+
+```
+php artisan rapidez-statamic:install
+```
+
+Please make sure to use the following settings when setting up the eloquent driver:
```

Comment:
What are the settings? Maybe add a stub with a pre configured config file?

Reply:
Because of changing config files from the eloquent driver i would rather document what to select for eloquent here rather than having an outdated config file.